src/platform/mhv/secondary-nav/tests/MhvSecondaryNavMenu.unit.spec.jsx

```diff
@@ -56,64 +44,31 @@ const setWindowUrl = pathname => {
 };
 
 describe('MHV Secondary Navigation Menu Component', () => {
-  describe('feature toggle', () => {
-    it('renders when the toggle is on', () => {
-      const mock = mockStore({ isFeatureEnabled: true });
-      const { getAllByRole } = render(
-        <Provider store={mock}>
-          <MhvSecondaryNavMenu items={testSecNavItems} />
-        </Provider>,
-      );
-      const links = getAllByRole('link');
-      expect(links.length).to.eql(testSecNavItems.length);
-      testSecNavItems.forEach((navItem, i) => {
-        expect(links[i].pathname).to.be.eql(navItem.href);
-      });
-    });
+  const activeClassString = 'active';
+  const medNavItem = {
+    title: 'Medications',
+    abbreviation: 'Meds',
+    icon: 'fas fa-prescription-bottle',
```

Comment:
The design system is actively migrating font awesome to `va-icon`. Would it be possible to update the usage of this font awesome class name, and whatever element it maps to with `va-icon`? It [looks like](https://design.va.gov/foundation/icons#profile) the equivalent icon is called `medication`.

Reply:
Yes, we can change this, but note this is only a string at this point in the test and not used to display any real icons. We can do that in a different PR to get this out the door.